### PR TITLE
Successful Sign-In from Non-Compliant Device with Bulk Download Activity

### PR DESCRIPTION
**Pull request by gitj121**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Hunting Queries/MultipleDataSources/NonCompliantSigninwithBulkDownload.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  AuditLogs
+  | where OperationName has_any  ("Download group members", "Download users", "Download user registeration details", "Download groups")
+  | extend IpAddress = case(
+  isnotempty(tostring(parse_json(tostring(InitiatedBy.user)).ipAddress)) and tostring(parse_json(tostring(InitiatedBy.user)).ipAddress) != 'null', tostring(parse_json(tostring(InitiatedBy.user)).ipAddress),
+  isnotempty(tostring(parse_json(tostring(InitiatedBy.app)).ipAddress)) and tostring(parse_json(tostring(InitiatedBy.app)).ipAddress) != 'null', tostring(parse_json(tostring(InitiatedBy.app)).ipAddress),'Not Available')
+  | extend InitiatedBy = iff(isnotempty(tostring(parse_json(tostring(InitiatedBy.user)).userPrincipalName)),
+    tostring(parse_json(tostring(InitiatedBy.user)).userPrincipalName), tostring(parse_json(tostring(InitiatedBy.app)).displayName)), UserRoles = tostring(parse_json(tostring(InitiatedBy.user)).ipAddress)
+  | extend TargetResourceName = tolower(tostring(TargetResources.[0].displayName))  
+  | project TimeGenerated, InitiatedBy, CorrelationId, Id,AADTenantId, ResourceId, IpAddress, OperationName
+  ) on $left.UserPrincipalName == $right.InitiatedBy
```
Comment (petebryan) [inline Hunting Queries/MultipleDataSources/NonCompliantSigninwithBulkDownload.yaml:36]: Some times UPN and InitiatedBy can be capitalized differently. You might want to project these with tolower() in order to ensure you are getting the matched you expect.

File: Hunting Queries/MultipleDataSources/NonCompliantSigninwithBulkDownload.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  | project TimeGenerated, UserPrincipalName, IPAddress, CorrelationId, Id, ResourceId, OperationName
+  | extend AccountCustomEntity = UserPrincipalName
+  | extend IPCustomEntity = IPAddress
+    entityMappings:
```
Comment (petebryan) [inline Hunting Queries/MultipleDataSources/NonCompliantSigninwithBulkDownload.yaml:40]: Looks like an incorrect indent on this line

**Review by petebryan — CHANGES_REQUESTED** [review]

File: Hunting Queries/MultipleDataSources/NonCompliantSigninwithBulkDownload.yaml (hunk 1 above)
Comment (gitj121) [reply Hunting Queries/MultipleDataSources/NonCompliantSigninwithBulkDownload.yaml:38]: Thanks for the review, Pete. Made changes as requested.

File: Hunting Queries/MultipleDataSources/NonCompliantSigninwithBulkDownload.yaml (hunk 2 above)
Comment (gitj121) [reply Hunting Queries/MultipleDataSources/NonCompliantSigninwithBulkDownload.yaml]: Resolved in the new commit

**Review by aprakash13 — APPROVED** [review]
Thanks

**Review by shainw — APPROVED** [review]

**Pull request merged**